Sources/SwiftParser/Types.swift

```diff
@@ -689,13 +689,28 @@ extension Parser.Lookahead {
 
   mutating func skipTypeAttributeList() {
     var specifierProgress = LoopProgressCondition()
-    // TODO: Can we model isolated/_const so that they're specified in both canParse* and parse*?
     while canHaveParameterSpecifier,
-      self.at(anyIn: SimpleTypeSpecifierSyntax.SpecifierOptions.self) != nil || self.at(.keyword(.isolated))
-        || self.at(.keyword(._const)),
+      self.at(anyIn: SimpleTypeSpecifierSyntax.SpecifierOptions.self) != nil
+        || self.at(.keyword(.nonisolated), .keyword(.dependsOn)),
       self.hasProgressed(&specifierProgress)
     {
-      self.consumeAnyToken()
+      switch self.currentToken {
+      case .keyword(.nonisolated), .keyword(.dependsOn):
+        self.consumeAnyToken()
+
+        // The argument is missing but it still could be a valid modifier,
+        // i.e. `nonisolated` in an inheritance clause.
+        guard self.at(.leftParen), !self.currentToken.isAtStartOfLine else {
```

Comment:
This can be unified slightly to 

```suggestion
        guard self.at(TokenSpec(.leftParen, allowAtStartOfLine: false)) else {
```

Reply:
Nice, thank you!

Reply:
✅